From 9380ff5a8849ea5a772958b5dcfc5462182f1597 Mon Sep 17 00:00:00 2001
From: Yousong Zhou <yszhou4tech@gmail.com>
Date: Fri, 20 Mar 2020 17:06:31 +0800
Subject: [PATCH] build: skip tests

Tests build can fail for arc700 for the following causes

 - Missing -stack-protector-all argument when building test progs
 - Missing linker flag -lgmodule-2.0 -lffi

Signed-off-by: Yousong Zhou <yszhou4tech@gmail.com>
---
 server/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/server/Makefile.am b/server/Makefile.am
index 2d148c14..5b927a1b 100644
--- a/server/Makefile.am
+++ b/server/Makefile.am
@@ -1,5 +1,5 @@
 NULL =
-SUBDIRS = . tests
+SUBDIRS = .
 
 check-valgrind:
 	$(MAKE) -C tests check-valgrind
